### PR TITLE
 CQ-4307825 Inclusion of forms addon in aem archetype

### PR DESCRIPTION
**Pull request by rismehta**
- Inclusion of forms core components
            - Inclusion of forms sdk
            @review @vdua
            DOD(Yes)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by rismehta** [comment]
Can someone review this ? On analysing the logs, it looks like `windows-latest, 11` job failed due to connection time out on `repo.adobe.com`. I have re-triggered the build.

**Review by bpauli — COMMENTED** [review]
LGTM. But I wonder if there should also a proxy component for the `AEM Forms Container (v1)` in ui.apps in case the forms flag is enabled.

**Comment by rismehta** [comment]
@bpauli I was planning to add the proxy components, its corresponding template policies etc in next iteration, I have already have the CL ready, https://github.com/rismehta/aem-project-archetype/commit/4bbd738d1af50541a404f5d1d1e049c3bb9df536. If this CL looks good, can you merge the PR, since I don't have merge rights yet.

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/aemformscontainer/.content.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bpauli) [inline src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/aemformscontainer/.content.xml:1]: This proxy component should be deleted when the forms flag is not set.

**Review by bpauli — CHANGES_REQUESTED** [review]
The proxy components should be deleted when the form flag is not set.

File: src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/aemformscontainer/.content.xml (hunk 1 above)
Comment (rismehta) [reply src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/aemformscontainer/.content.xml:1]: thanks for the review @bpauli. I have added this logic.

**Review by bpauli — APPROVED** [review]

**Pull request merged**